sdk/identity/azure-identity/azure/identity/_enums.py

```diff
@@ -12,7 +11,7 @@ class RegionalAuthority(str, Enum, metaclass=CaseInsensitiveEnumMeta):
 
     #: Attempt to discover the appropriate authority. This works on some Azure hosts, such as VMs and
     #: Azure Functions. The non-regional authority is used when discovery fails.
-    AUTO_DISCOVER_REGION = ConfidentialClientApplication.ATTEMPT_REGION_DISCOVERY
+    AUTO_DISCOVER_REGION = "tryautodetect"
```

Comment:
Question: It doesn't appear `RegionalAuthority` is exported. Was the intent for users to use the `AUTO_DISCOVER_REGION` when setting `AZURE_REGIONAL_AUTHORITY_NAME`? Or is the typical usage pattern like:

```bash
export AZURE_REGIONAL_AUTHORITY_NAME=true
export AZURE_REGIONAL_AUTHORITY_NAME=TryAutoDetect
```

Reply:
Exposing the consts are not in the scope of this PR. :)